kentik_api_library/kentik_api/public/device.py

```diff
@@ -43,13 +42,40 @@ class DeviceSubtype(Enum):
     ibm_subnet = "ibm_subnet"  # not in api documentation
 
 
-class DeviceBGPType(Enum):
-    none = "none"
-    device = "device"
-    other_device = "other_device"
+# class DeviceBGPType(Enum):
+#     none = "none"
+#     device = "device"
+#     other_device = "other_device"
+
+DEVICE_BGP_TYPE__NONE = "none"
```

Comment:
I modified 2 enums with in 2 different ways:
1. DeviceBGPType substituted with group of "constants".
2. CDNAttribute based on modified metaclass.

There are also other solution described here: https://stackoverflow.com/questions/55996561/support-unknown-values-in-python-enums

I think that the solution number 2. (CDNAttribute) is the best one, because it works good enough and does not need external libraries. @mateuszmidor @lukaa12 What do you think?